File: src/main/java/com/gitlab/daring/image/swing/NotificationUtils.java
package com.gitlab.daring.image.swing;

import javax.swing.*;
import java.awt.*;

import static javax.swing.JOptionPane.ERROR_MESSAGE;
import static javax.swing.JOptionPane.showMessageDialog;

public class NotificationUtils {

    public static void showErrorDialog(Component c, String msg) {
        JTextArea field = new JTextArea(10, 40);
        field.setLineWrap(true);
        field.setEditable(false);
        field.setText(msg);
        showMessageDialog(c, new JScrollPane(field), "Ошибка", ERROR_MESSAGE);
    }

    private NotificationUtils() {
    }

}

File: src/main/kotlin/com/gitlab/daring/image/swing/NotificationUtils.kt
package com.gitlab.daring.image.swing

import java.awt.Component
import javax.swing.JOptionPane.ERROR_MESSAGE
import javax.swing.JOptionPane.showMessageDialog
import javax.swing.JScrollPane
import javax.swing.JTextArea

object NotificationUtils {

    @JvmStatic
    fun showErrorDialog(c: Component, msg: String) {
        val field = JTextArea(10, 40)
        field.lineWrap = true
        field.isEditable = false
        field.text = msg
        showMessageDialog(c, JScrollPane(field), "Ошибка", ERROR_MESSAGE)
    }

}